### ISSUE TITLE
Move VRP to being a sandbox (only)

### ISSUE THREAD

**Issue by phlax** [envoyproxy/envoy#26422]
Currently we re/build the VRP image/s on every CI run and publish them to dockerhub

This takes up a lot of CI cycles/time

Mostly i think due to lack of documentation, the VRP images dont seem to get a lot of air play

There are other issues with how the images are currently built that cause headaches/workarounds for the sandboxes - where VRP has 2 despite lacking docs

My suggestion is to shift what is currently being built into an image into a sandbox recipe

If someone wants to use the image they can build/customize it quickly from the sandbox where it will have documentation explaining how to build and use it, some of the tools included and ways to make use of them

I think doing this will kill 3 proverbial birds with 1 stone - ie speedup/simplify docker ci/publishing, make the VRP more visible with better docs, and provide a better resolution to the current sandbox workarounds

**Comment by phlax** [comment]
as a case in point ive again spent a good part of today making the vrp build/sandboxes work with everything else

**Comment by htuch** [comment]
Yeah, totally welcome improvement if you want to do this, thanks.
